pyroma/tests.py

```diff
@@ -169,7 +169,6 @@ def test_minimal(self):
             (
                 2,
                 [
-                    "The package's version number does not comply with PEP-386 or PEP-440.",
```

Comment:
Did this become untestable now? I know setuptools is getting stricter with it.

Reply:
I see from a different issue, that yes, that's exactly what happened.

Reply:
Yup!

Also see OP:

> EDIT 4: Fix test failures with Setuptools >66.0.0 due to invalid versions making packages unbuildable (see https://github.com/regebro/pyroma/issues/95)